packages/lab/src/radio-button/RadioButton.css

```diff
@@ -1,57 +1,35 @@
-/* Styles applied to root component  */
-.salt-density-medium {
-  --radioButton-label-marginTop: 6px;
-  --radioButton-row-marginRight: 24px;
-}
-
-.salt-density-high {
-  --radioButton-label-marginTop: 3px;
-  --radioButton-row-marginRight: 12px;
-}
-
-.salt-density-low {
-  --radioButton-label-marginTop: 9px;
-  --radioButton-row-marginRight: 36px;
-}
-
-.salt-density-touch {
-  --radioButton-label-marginTop: 12px;
-  --radioButton-row-marginRight: 48px;
-}
-
+/* Styles applied to RadioButton container */
 .saltRadioButton {
+  display: flex;
+  gap: calc(var(--salt-size-unit) * 0.75);
+  position: relative;
+  user-select: none;
   cursor: pointer;
-  display: block;
-}
-
-/* Styles applied to root component on hover if `disabled={true}` or FormField disabled */
-.saltRadioButton-disabled,
```

Comment:
you can do the same with `.saltRadioButton:disabled` and save a class

Reply:
saltRadioButton is a label element and doesn't take a disabled argument

Reply:
aaah i see, the input is inside x.x... makes sense :)
one day, whenever firefox wants, we will be able to do `SRB:has(:disabled)` 😍 ... for now I'll keep dreaming XD